hawc/apps/common/backends.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth.backends import BaseBackend
```

Comment:
rename this file to `auth` instead of `backends`; backends could refer to other things such as db or cache